jgiganto/MorseToLight/readme.md

```diff
@@ -0,0 +1,201 @@
+# MORSE TO LIGHT
+Muy buenas, en esta ocasión vamos a aprender a realizar una app con Xamarin para dispositivos Android. Básicamente la función de nuestra app será traducir un texto a código morse y transmitirlo usando para ello el flash de la cámara de nuestro smartphone.
+Para ello vamos a apoyarnos en la API multiplataforma Xamarin Essentials, la cual permite hacer uso de los sensores del dispositivo (giroscopio, brújula, etc..) de una manera sencilla.
+Comenzamos:
+Abrimos un nuevo proyecto en Visual Studio. En este caso nuestro proyecto será del tipo Xamarin.forms como se indica en la imagen.
+
+
+![image1](images/image1.png)
+
+Además vamos a instalar el paquete Nugget Xamarin Essentials, que nos permitirá hacer uso del flash de la cámara.
+
+![image2](images/image2.png)
+
+Para ello es importante que marquemos la casilla Include prerelease, ya que al ser una versión pre-lanzamiento si no la marcamos no nos aparecerá.
```

Comment:
En el caso de nombrar "Include prerelease" lo marcaría en negrita o entre comillas

Reply:
[Xamarin Essentials](https://www.nuget.org/packages/Xamarin.Essentials/) is not in pre-release anymore.